src/dbos.ts

```diff
@@ -1371,26 +1370,49 @@ export class DBOS {
     return DBOS.#executor.getEvent(workflowID, key, timeoutSeconds);
   }
 
+  /**
+   * registers a workflow method or function with an invocation schedule
+   * @param func - The workflow method or function to register with an invocation schedule
+   * @param options - Configuration information for the scheduled workflow
+   */
+  static registerScheduled<This, Return>(
+    func: (this: This, ...args: ScheduledArgs) => Promise<Return>,
+    crontab: string,
```

Comment:
Why is this not a SchedulerConfig?

Do you call this on the workflow or on the registered workflow? The examples show the latter, but the options look like those for the former.

Reply:
AFAIK, you call this on the registered workflow. That was the way SQS and kafka receivers did it, but it may  changing in #989.

Reply:
Yeah, if you call it on the registered workflow, then these parameters don't all make sense (it already has a name, etc.)

Reply:
I'm going to change the signature to func / schedulerConfig / targetConfig in this PR. As per @chuck-dbos's [comment](https://github.com/dbos-inc/dbos-transact-ts/pull/989#issuecomment-3046427577), we need a pass on standarizing config / target parameters in all register functions, but we'll handle that as a separate PR